packages/appstore/src/components/modals/cashier-wallet-modal/cashier-wallet-modal-header.tsx

```diff
@@ -0,0 +1,139 @@
+        if (is_demo) return 'Demo';
+        if (['svg', 'malta'].includes(shortcode)) return shortcode.toUpperCase();
+        return '';
+    }, [is_demo, shortcode]);
+
+    const getCloseIcon = React.useCallback(() => {
+        if (is_demo && is_dark) return 'IcAppstoreCloseLight';
+        if (is_demo && !is_dark) return 'IcAppstoreCloseDark';
+        if (is_dark) return 'IcAppstoreCloseDark';
+        return 'IcAppstoreCloseLight';
+    }, [is_dark, is_demo]);
+
+    const getStylesByClassName = React.useCallback(
+        (class_name: string) => {
+            return classNames(class_name, {
+                [`${class_name}-demo`]: is_demo,
+            });
+        },
+        [is_demo]
+    );
+
+    const getCurrencyIconSize = React.useCallback(() => {
+        // TODO: add p2p and payment_agent check
+        const is_square_icon =
+            ['btc', 'eth', 'ltc', 'usdt', 'eusdt', 'tusdt', 'ust', 'usdc', 'p2p', 'payment_agent'].includes(
+                currency.toLowerCase()
+            ) || is_demo;
+
+        const sizes = {
+            mobile: {
+                width: is_square_icon ? 64 : 48,
+                height: is_square_icon ? 40 : 48,
+            },
+            desktop: {
+                width: is_square_icon ? 128 : 64,
+                height: is_square_icon ? 80 : 64,
+            },
+        };
+
+        const size = is_mobile ? sizes.mobile : sizes.desktop;
+
+        return size;
+    }, [currency, is_demo, is_mobile]);
+
+    const getCurrencyIconProps = React.useCallback(() => {
+        const icon = getWalletCurrencyIcon(is_demo ? 'demo' : currency, is_dark);
+        const size = getCurrencyIconSize();
+
+        return { icon, ...size };
+    }, [currency, getCurrencyIconSize, is_dark, is_demo]);
+
+    const getBackgroundName = React.useCallback(() => {
+        if (is_demo) {
+            return 'demo';
+        }
+        return currency.toLowerCase();
+    }, [is_demo, currency]);
+
+    return (
+        <GradientBackground {...getCashierWalletModalBackgrounds(getBackgroundName())}>
+            {is_demo && <Watermark image={`url(${is_dark ? DemoDark : DemoLight})`} opacity={is_dark ? 0.32 : 0.24} />}
+            <div
+                className={classNames(header_class_name, {
+                    [`${header_class_name}--hide-title`]: !show_wallet_name,
+                })}
+            >
+                <div className={`${header_class_name}__title-wrapper`}>
+                    {show_wallet_name && (
+                        <div className={classNames(`${header_class_name}__title`)}>
+                            <Text
+                                size={is_mobile ? 'xs' : 's'}
+                                as='span'
+                                className={getStylesByClassName(`${header_class_name}__title-wallet`)}
+                            >
+                                {is_demo ? 'Demo' : ''} {currency} {localize('Wallet')}
```

Comment:
Do we need to localize `Demo` word? I think yes. In this case we need to use `Localize` component